modules/oe_translation_poetry/tests/Functional/PoetryTranslationRequestTest.php

```diff
@@ -344,4 +652,22 @@ protected function assertJobsPoetryRequestIdValues(array $jobs, array $values):
     }
   }
 
+  /**
+   * Groups the jobs by their target language.
+   *
+   * @param \Drupal\tmgmt\Entity\JobInterface[] $jobs
+   *   The jobs.
+   *
+   * @return \Drupal\tmgmt\Entity\JobInterface[]
+   *   The jobs.
+   */
+  protected function groupJobsByLanguage(array $jobs): array {
```

Comment:
Nickpicking here, but the method should be named indexJobsByLanguage since we are not grouping multiple jobs under any given language but using the language of the jobs as the keys for the array.

Reply:
OK, will do.